tests/40-sd-create.js

```diff
@@ -127,14 +130,33 @@ describe('ecdsa-sd-2023 (create)', function() {
                 '"proofValue" property that starts with "u".'
               );
             });
-            it('The "proof" MUST verify when using a conformant verifier.',
+            it('The "proof" MUST verify with a conformant verifier.',
               async function() {
                 this.test.cell = {
                   columnId: `${name}: ${keyType}`, rowId: this.test.title
                 };
-                should.exist(verifier, 'Expected implementation to have a VC ' +
-                  'HTTP API compatible verifier.');
-                verificationSuccess({credential: issuedVc, verifier});
+                // First we need to derive a credential
+                const {purposes: {AssertionProofPurpose}} = jsigs;
+                const suite = new DataIntegrityProof({
+                  // disclose only the `credentialSubject.id` path
+                  cryptosuite: ecdsaSd2023Cryptosuite
+                    .createDiscloseCryptosuite({
+                      selectivePointers: [
+                        '/credentialSubject/id'
+                      ]
+                    })
+                });
+
+                const derivedCredential = await jsigs.derive(issuedVc, {
```

Comment:
there is an existing deriveCredential function in `vc-generator` used for the sd-verify test data.

Reply:
also might want to add a comment about how anyone can derive a proof from a base proof.